styles/_header.scss

```diff
@@ -42,13 +72,21 @@ $header-height: 71px;
       display: inline-block;
     }
 
-    border: 0;
-    color: $link_color;
+    color: $primary;
     display: none;
-    font-size: 14px;
-    padding: 0 20px 0 0;
+    font-size: 0.8em;
+    font-family: $font--heading;
+    font-weight: $font__weight--black;
+    letter-spacing: 1px;
+    text-transform: uppercase;
+    padding: 0 16px 0 0;
```

Comment:
variables for all these 16pxs?

Reply:
Absolutely if it turns out to be a repeatable value! My next PR should finalize the style for the header.